velox/common/caching/tests/CachedFactoryTest.cpp

```diff
@@ -205,51 +226,21 @@ TEST(CachedFactoryTest, retrievedCached) {
       std::make_unique<SimpleLRUCache<int, int>>(1000), std::move(generator));
   for (int i = 0; i < 10; i += 2)
     factory.generate(i);
-  ASSERT_EQ(*generated, 5);
+  EXPECT_EQ(*generated, 5);
   std::vector<int> keys(10);
   for (int i = 0; i < 10; i += 1)
     keys[i] = i;
-  std::vector<std::pair<int, CachedPtr<int, int>>> cached;
+  std::vector<std::pair<int, int>> cached;
   std::vector<int> missing;
   factory.retrieveCached(keys, &cached, &missing);
   ASSERT_EQ(5, cached.size());
   for (int i = 0; i < 5; ++i) {
-    ASSERT_EQ(cached[i].first, 2 * i);
-    ASSERT_EQ(*cached[i].second, 4 * i);
-    ASSERT_TRUE(cached[i].second.wasCached());
+    EXPECT_EQ(cached[i].first, 2 * i);
+    EXPECT_EQ(cached[i].second, 4 * i);
   }
   ASSERT_EQ(5, missing.size());
   for (int i = 0; i < 5; ++i) {
-    ASSERT_EQ(missing[i], 2 * i + 1);
+    EXPECT_EQ(missing[i], 2 * i + 1);
   }
-  ASSERT_EQ(*generated, 5);
-}
-
-TEST(CachedFactoryTest, clearCache) {
```

Comment:
Why remove this test?

Reply:
Let's keep the test in CachedFactoryTest.cpp in case that we break this contract with SimpleLRUCache by mistake if it is not too hard? Thanks!